docs/index.md

```diff
@@ -17,18 +17,16 @@ Incorporating DID.app gives your users a fast, easy, and safe way to sign in usi
 
 Follow our step-by-step [integration guide](/docs/step-by-step-integration) to get started.
 
-
-
-<!-- New to DID.app? Follow one of our implementation guides
+Or try one of our specific platform guides.
 
 - [Phoenix & Elixir](/guides/phoenix-elixir-openid-connect-integration)
 - [Express & Node.js](/guides/express-nodejs-openid-connect-integration)
 - [Wordpress](/guides/wordpress-openid-connect-client-plugin-integration/)
 
-Can't find what your platform, the simplest way to integrate with DID is using an OpenID Connect library for your platform.
+<!-- Can't find what your platform, the simplest way to integrate with DID is using an OpenID Connect library for your platform.
```

Comment:
```suggestion
<!-- Can't find your platform? The simplest way to integrate with DID is using an OpenID Connect library for your platform.
```

Reply:
You probably can ignore the commented out stuff